src/scripts/release.zig

```diff
@@ -159,7 +161,6 @@ fn build_tigerbeetle(shell: *Shell, info: VersionInfo, dist_dir: std.fs.Dir) !vo
                 \\    -Dtarget={target}
                 \\    -Drelease={release}
                 \\    -Dgit-commit={commit}
-                \\    -Dversion={version}
             , .{
                 .target = target,
                 .release = if (debug) "false" else "true",
```

Comment:
`version` (a few lines down) can be removed

Reply:
Even must be, shell checks for unusead args!

Reply:
removed